pkg/api/customization/logging/action.go

```diff
@@ -215,14 +221,35 @@ func (h *Handler) dryRunLoggingTarget(apiContext *types.APIContext, level, clust
 	if err != nil {
 		return errors.Wrap(err, "create executor failed")
 	}
-	var stdout, stderr, stdin bytes.Buffer
+	mkdirCmd := bytes.NewBufferString("mkdir -p " + sslDir)
```

Comment:
You need to considerate multiple dryRun actions run at the same time. You need to create a tmp dir for each action at least.

Reply:
The fluentd command is ```fluentd -q --dry-run -i ```  ```-i``` means inline, we don't create the file, inline config just appended to the config file on-fly.

Reply:
add generate temp dir for cert every time